clang/test/CheckedCRewriter/arrcallee.c

```diff
@@ -1,7 +1,10 @@
-// RUN: cconv-standalone -alltypes %s -- | FileCheck -match-full-lines -check-prefixes="CHECK_ALL","CHECK" %s
-//RUN: cconv-standalone %s -- | FileCheck -match-full-lines -check-prefixes="CHECK_NOALL","CHECK" %s
-// RUN: cconv-standalone %s -- | %clang -c -fcheckedc-extension -x c -o /dev/null -
+// RUN: cconv-standalone -alltypes -addcr %s -- | FileCheck -match-full-lines -check-prefixes="CHECK_ALL","CHECK" %s
+// RUN: cconv-standalone -addcr %s -- | FileCheck -match-full-lines -check-prefixes="CHECK_NOALL","CHECK" %s
+// RUN: cconv-standalone -addcr %s -- | %clang -c -fcheckedc-extension -x c -o /dev/null -
 
+// RUN: cconv-standalone -alltypes -output-postfix=checked %s
```

Comment:
You are running `cconv-standalone` twice for `-alltypes` -- one time you pipe to `stdout` but then you send it to a file and do a diff. Why not just send it to a file at the start, and then do `FileCheck` on that file, rather than using `stdout` ?

Reply:
Also, why are we only checking re-convertability for the `-alltypes` cases, not not for the plain cases too (i.e., without `-alltypes` enabled) ?

Reply:
1) You're right, we can shave off a conversion, so for the first comment, I can change that to just happen once 
2) I figured that running with alltypes is the same as running without alltypes, but just with more conversions. So whenever we run the tool, it seems we do all the analysis for array pointers anyway and just make it all wild if it turns out the user didn't pass in alltypes, so I thought we'd get the most out of the "reconversion" if we had more checked types to play with. I can add the non-alltypes case too though if we'd like (didn't at first because it seemed like a lot of stuff was getting added to the tests).

Reply:
> You are running `cconv-standalone` twice for `-alltypes` -- one time you pipe to `stdout` but then you send it to a file and do a diff. Why not just send it to a file at the start, and then do `FileCheck` on that file, rather than using `stdout` ?

Except that one has `-addcr` and the other doesn't. So perhaps leave things as they are? Same for #2. I think it's fine as is.